capabilities/learning-culture.md

```diff
@@ -1,85 +1,105 @@
+Talking directly with users is an invaluable practice for gaining insights and understanding their needs and challenges. Field customer support calls. Host developer office hours. Run focus groups. Whatever method you use, engaging with users helps to gather real-world feedback, identify pain points, and uncover opportunities for improvement. By maintaining direct communication with users, you can ensure that your product or service aligns closely with expectations and foster a stronger connection between your team and your user base.
 
 ### [Dogfood Your Systems](/practices/dogfood-your-systems.md)
 
-Dogfooding your systems involves having your teams use the same products or systems that your users do, allowing them to experience the same pain points firsthand. This practice helps build empathy with users, identify issues early, and drive improvements based on direct user experience. By regularly using your own systems, your team can gain valuable insights and ensure that the product meets the highest standards of usability and performance.
+Dogfooding your systems involves having your teams use the same products or systems that your users do, allowing them to experience the same pain points firsthand. This practice helps build empathy with users, identify issues early, and drive improvements based on direct "user" experience. By regularly using your own systems as customers would, your team can gain valuable insights and ensure that the product meets the highest standards of usability and performance.
 
-### [Start A Community Of Practice](/practices/start-a-community-of-practice.md)
+### [Start a Community Of Practice](/practices/start-a-community-of-practice.md)
 
-Starting a community of practice provides a dedicated space where a variety of practices can be performed. This collaborative environment fosters continuous learning, sharing of best practices, and collective problem-solving. By bringing together practitioners with shared interests, a community of practice can drive innovation, enhance skills, and promote a culture of continuous improvement. It serves as a flexible platform for implementing various practices and exploring new ideas within the group.
+Starting a community of practice means providing a dedicated space where a variety of practices can be performed. This collaborative environment fosters continuous learning, sharing of best practices, and collective problem-solving. By bringing together practitioners with shared interests, a community of practice can drive innovation, enhance skills, and promote a learning culture. 
```

Comment:
What would such a collaborative environment actually look like, practically? A weekly/monthly meeting in the workplace where a group can implement some of the practices on this list (book club, viewing party, demonstration)?

Reply:
From my experience, they've usually been monthly. But there's no hard and fast rule on that per se.